### PR TITLE
Update developer guide

### PR DESCRIPTION
**Pull request by lesterong**
- Update links for developer guide

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -545,15 +545,16 @@ testers are expected to do more *exploratory* testing.
 
 1. Initial launch
 
-   1. Download the jar file and copy into an empty folder
+   1. Download the jar file and copy into an empty folder.
```
Comment (ichigh0st) [inline docs/DeveloperGuide.md:548]: should we change the numbering to 1a, 1b, etc.?

**Review by ichigh0st — APPROVED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (lesterong) [reply docs/DeveloperGuide.md:548]: We could, but the formatting will look like this instead. 

image

For reference, the current formatting is as such
image

Alternatively, we can write CSS to account or that, or just ignore it. Thoughts? @ichigh0st

Comment (ichigh0st) [reply docs/DeveloperGuide.md:557]: ok then nvm css seems like a lot of effort

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -502,7 +511,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
 
-* 1b. The given profile to add is invalid
+* 1b. The given profile to add is invalid.
 
     * 1b1. NUScheduler shows an error message.
```
Comment (ichigh0st) [inline docs/DeveloperGuide.md:517]: for line 529 below should we change to 1000 by 700px? alternatively can change min size of MainWindow in fxml as well.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (lesterong) [reply docs/DeveloperGuide.md:557]: The CSS will look something like the following, not too difficult if its what the preferred behaviour. 😄 

```html
<div style="display: flex; gap: 4px">
<span>1b.</span>
Re-launch the app by double-clicking the jar file.<br>
Expected: The most recent window size and location is retained.
</div>

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (lesterong) [reply docs/DeveloperGuide.md:518]: I agree. Let's update Line 529 instead, changing the window size is probably an enhancement.

**Pull request merged**